### ISSUE TITLE
Links: Add hover and underline

### ISSUE THREAD

**Issue by sarahfossheim** [oslopride/oslopride.no#142]
All links, both in body text and footer, should be underlined and have a hover effect. 

Footer:
- Suggestion is to use `#E350A0` as the hover color (https://github.com/oslopride/oslopride.no/issues/126). _(not accessible with regards to color contrast)_

Body text:
- Suggestion is to use `#3a1b7b` as the link color (https://github.com/oslopride/oslopride.no/issues/126).

**Comment by hexjelly** [comment]
Should the menu links also be underlined?

**Comment by sarahfossheim** [comment]
I think for the menu items the border thing on the left should be good enough. @lovise-hub what do you think?

**Comment by lovise-hub** [comment]
No need to put border on menu/navigation items. @sarahfossheim @hexjelly

**Comment by hexjelly** [comment]
I'll just close this because it seems like #156 and #159 supersedes it.
